src/components/pages/Books.js

```diff
@@ -10,18 +10,20 @@ const Books = () => {
   useEffect(() => {
     dispatch(displayBooks());
   }, []);
+  const bookList = books.map((book) => (
+    <Book
+      key={book.id}
+      id={book.id}
+      author={book.author}
+      title={book.title}
+      category={book.category}
+    />
+  ));
 
```

Comment:
### *NOTE: The changes below are not related to this line. I used it to add an inline comment. Please refer to the comments below to gain clarity on the changes you should implement.* :pray: 
---


- [ ] Please, kindly add the following missing part of the design below to make your design close to the given template

*your design*
![Screenshot from 2022-11-10 11-20-27](https://user-images.githubusercontent.com/59397159/201066966-6ce80160-2b11-40a2-a4ce-d3eba9d32c67.png)
*original design*
![Screenshot from 2022-11-10 11-19-06](https://user-images.githubusercontent.com/59397159/201066973-8647dd29-1361-4281-906f-ef7f54e843bc.png)

*missing icon*
![Screenshot from 2022-11-10 11-17-10](https://user-images.githubusercontent.com/59397159/201066976-22e33fac-10df-40d0-837e-32e11068e9d1.png)

---

- Please, kindly use the correct color here:

*color from design*
![Screenshot from 2022-11-10 11-16-47](https://user-images.githubusercontent.com/59397159/201066978-56eaed33-eb97-4eef-8d1c-6bacc6541ec4.png)

*your design*
![Screenshot from 2022-11-10 11-32-48](https://user-images.githubusercontent.com/59397159/201068149-a38d482d-f6b3-4713-87e9-98539a0eb69f.png)